doc/examples/tutorial/example11.json

```diff
@@ -0,0 +1,42 @@
+{
+	/*
+	 * Use case which creates a thread with 3 phases. In each phase the thread
+	 * runs for 2ms and then sleeps for 8ms. In phase0 the thread runs in
+	 * taskgroup /tg1/tg11. Phase1 does not specify a taskgroup so the thread
+	 * will continue to run in taskgroup /tg1/tg11. In phase2 the thread will
+	 * run in taskgroup / (the root taskgroup). The use case runs for 2secs or
+	 * until it is stopped with Ctrl+C.
+	 */
+	"tasks" : {
+		"thread0" : {
+			"loop" : -1,
+			"phases" : {
+				"phase0" : {
+					"run" : 20000,
+					"sleep" : 80000,
+					"taskgroup" : "/tg1/tg11"
```

Comment:
@deggeman you've decided to make taskgroup changes operate the same as a sched policy - which means a phase can only have one, and it takes effect as soon as entering a phase/thread. I think this is maybe a problem with our json grammer, but it's not that clear to me that this is what will happen when I add one of these to a phase (we could perhaps make the documentation more explicit that it doesn't work like an event?)

Reply:
From the json grammer perspective, "taskgroup" behaves like "cpus" (cpuset) or "policy/priority (sched_data). The latter ones, you can also specify per thread/phase but they aren't events.

Reply:
:+1: